administration/src/util/qrcode.ts

```diff
@@ -246,3 +248,20 @@ export const drawQRCode = (
     size
   )
 }
+
+export const convertProtobufToHexCode = (protobufQRCode: DynamicActivationCode): string => {
```

Comment:
we also need the `StaticVerificationCode` to be converted because it will also be printed on the pdf by DATEV